praw/models/reddit/wikipage.py

```diff
@@ -214,6 +263,32 @@ def _fetch(self):
         self.__dict__.update(data)
         self._fetched = True
 
+    def discussions(
+        self, **generator_kwargs: Any
+    ) -> Iterator["praw.models.Submission"]:
```

Comment:
```suggestion
    ) -> Generator["praw.models.Submission", None, None]:
```
This should be generator.

Reply:
Ah yep you're right, my bad!